platform/viewer/src/googleCloud/ProjectsList.js

```diff
@@ -65,7 +65,11 @@ class ProjectsList extends Component {
     }
     const body = (
       <tbody id="ProjectList">
-        {projects.filter(project => (project.name.toLowerCase().includes(filter.toLowerCase()) || filter=="")).map(this.renderTableRow)}
+        {
+          projects.filter(project => ( typeof project.name  === 'string' &&
+            (filter=="" || project.name.toLowerCase().includes(filter.toLowerCase()))
+          )).map(this.renderTableRow)
+        }
```

Comment:
Can you break this down outside of the render function so it's a bit easier to reason about?
Something like:

```js
const lowerCaseFilter = filter.toLowerCase();
const filteredProjects = projects.filter(project => 
    typeof project.name  === 'string' &&
    (filter === "" || project.name.toLowerCase().includes(lowerCaseFilter));

...

{ filteredProjects.map(this.renderTableRow) }
```

Reply:
not sure if got this right, i've moved filtering from body generation

Reply:
If you've tested this, then I'm happy to merge. Comment and let me know, and I'll pull the merge lever.

Reply:
sure, i've tested it, also have tested in docker image
also i may deploy on publicly available instance

Reply:
If the PR is coming from a branch off of `OHIF/Viewers`, you can approve this workflow and a test docker image will be published:

![image](https://user-images.githubusercontent.com/5797588/69759090-3fd0b900-112f-11ea-94f0-72685802735f.png)